UNRELEASED.md

```diff
@@ -10,6 +10,7 @@
 ### Enhancements
 
 - Added utilities for parsing video duration (https://polaris.shopify.com/components/images-and-icons/video-thumbnail) ([#2725](https://github.com/Shopify/polaris-react/pull/2725))
+- Removed max-height property from `Tooltip`.
```

Comment:
```suggestion
- Removed max-height property from `Tooltip` [#2908](https://github.com/Shopify/polaris-react/pull/2908).
```

Don't forget to link to this PR in here as well

Reply:
Done!